doc/changelog/06-Ltac2-language/20475-ltac2_expose_inductive.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  API functions for inductive types - get number of parameters of inductive type, number of recursively uniform parameters, number of arguments for each constructor, number of variables bound in a pattern match for each constructor, convert case to inductive
```

Comment:
I generally prefer to write the names of the APIs instead of some prose for the changelog (eg "`Ltac2.Constr.Case.inductive`" instead of "convert case to inductive", for multiple from the same module then eg "`foo`, `bar`, in `Ltac2.Ind`")
IMO if the names aren't terrible it's enough for readers to tell if they're interested, and then it's easier to find the function with its actual name than a description.